### PR TITLE
feat: Add functional test for pipeline token

### PR DESCRIPTION
**Pull request by ibu1224**
## Context

Currently, users can use the pipeline token.
However, no functional tests for feature.
Therefore, functional test is required for safe release.

## Objective
In this PR,  I add `feature file` of functional test for pipeline token.  

## References
https://github.com/screwdriver-cd/screwdriver/issues/614

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by sakka2** [comment]
Do we need to add the case that the test will fail when Calvin use pipeline token which does not belongs to him ?

**Comment by ibu1224** [comment]
@sakka2 
It seems better to add test case.
So I add test case.

**Comment by sakka2** [comment]
> @sakka2
> It seems better to add test case.
> So I add test case.

Thanks! LGTM

**Comment by ibu1224** [comment]
@jithine 
That’s true. 
I should have prevented  the test from running.
I add `@ignore` tag for each scenario. 
In this PR, I do not add `step definition`.
I add only `feature file` .

File: features/pipeline-token.feature [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@pipelinetoken
+Feature: Pipeline Token
+
+  A Pipeline token can be created and used in place of a password when
+    interacting with the regarding pipeline API. This would allow users to obtain valid JWTs and
```
Comment (tk3fftk) [inline features/pipeline-token.feature]: Is indentation wrong?

File: features/pipeline-token.feature [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      - Users can add description to their issued, so they know what they're using them for
+      - User can see when a token was last used
+      - Users can revoke a token at any time
+      - No one can ever look up the raw value after initially generating the token
+
+  Background:
+    Given: "calvin" is logged in
+  @ignore
+  Scenario: Generate New Pipeline Token
+    Given "calvin" dose not own a token named "tiger"
```
Comment (tk3fftk) [inline features/pipeline-token.feature]: I think pipeline token is owned by a pipleine, not a user.
"calivin" onws a pipeline, and the pipeline owns pipeline token.

File: features/pipeline-token.feature (hunk 1 above)
Comment (ibu1224) [reply features/pipeline-token.feature]: yes, I fix it.

File: features/pipeline-token.feature (hunk 2 above)
Comment (ibu1224) [reply features/pipeline-token.feature]: I think so.
I should fix wording.

File: features/pipeline-token.feature [hunk 3]
```diff
@@ -0,0 +1,64 @@
+@ignore
+@pipelinetoken
+Feature: Pipeline Token
+
+    A Pipeline token can be created and used in place of a password when
+    interacting with the regarding pipeline API. This would allow users to obtain valid JWTs and
+    interact with the Screwdriver API in a programmatic way.
+
+    Rules:
+        - Tokens should be specifically tied to the user that issued them
```
[the remark this answers is not in the document]
Comment (ibu1224) [reply features/pipeline-token.feature]: yes, I fix it.

File: features/pipeline-token.feature [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  Scenario: Generate New Pipeline Token
+    Given "calvin" created pipeline dose not own a token named "tiger"
+    When a new Pipeline token named "tiger" is generated
+    And the token is used to log in to the specific pipeline
+    Then a valid JWT is received that represents "calvin"
+    And the "tiger" token's 'last used' property is updated
+
+  Scenario: List Pipeline Tokens
+    Given "calvin" created pipeline owns an existing Pipeline token named "tiger"
+    When he lists all his tokens
```
[the remark this answers is not in the document]
Comment (ibu1224) [reply features/pipeline-token.feature]: I fix Scenario.

File: features/pipeline-token.feature [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        - No one can ever look up the raw value after initially generating the token
+
+  Background:
+    Given: "calvin" is logged in
+
+  Scenario: Generate New Pipeline Token
+    Given "calvin" created pipeline dose not own a token named "tiger"
+    When a new Pipeline token named "tiger" is generated
+    And the token is used to log in to the specific pipeline
+    Then a valid JWT is received that represents "calvin"
```
Comment (kumada626) [inline features/pipeline-token.feature]: The pipeline token does not tied to the user.
I think a valid JWT may not represent the user.

Comment (ibu1224) [reply features/pipeline-token.feature]: I confirm the contents of scenario.

**Comment by ibu1224** [comment]
@jithine 
Thank you review and approve.
I rebase it.

**Pull request merged**